### PR TITLE
Removed the support for the certificate manager.

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Review by ujjwal-ibm — APPROVED** [review]

File: website/docs/r/is_vpn_server.html.markdown [hunk 1]
```diff
@@ -48,7 +48,10 @@ Review the argument references that you can specify for your resource.
   **&#x2022;** For more information, about creating access tags, see [working with tags](https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#create-access-console).</br>
   **&#x2022;** You must have the access listed in the [Granting users access to tag resources](https://cloud.ibm.com/docs/account?topic=account-access) for `access_tags`</br>
   **&#x2022;** `access_tags` must be in the format `key:value`.
-- `certificate_crn` - (Required, String) The certificate CRN instance from Certificate Manager or CRN of secret from Secrets Manager for this VPN server. As the usage of certificate CRN from Certificate Manager is getting deprecated, it is recommended to use CRN of secret from Secrets Manager. 
+- `certificate_crn` - (Required, String) The certificate CRN of secret from Secrets Manager for this VPN server. 
+
+  !> **Removal Notification** Certificate Manager support is removed, please use Secrets Manager.
```
Comment (ujjwal-ibm) [inline website/docs/r/is_vpn_server.html.markdown:53]: why is it required when removed ?

File: website/docs/r/is_lb_listener.html.markdown [hunk 2]
```diff
@@ -188,7 +188,7 @@ Review the argument references that you can specify for your resource.
       - Not already be the default_pool for another listener 
 - `certificate_instance` - (Optional, String) The CRN of the certificate instance, it is applicable(mandatory) only to https protocol.
```
Comment (ujjwal-ibm) [inline website/docs/r/is_lb_listener.html.markdown:190]: if its removed then remove optional ?

**Review by ujjwal-ibm — CHANGES_REQUESTED** [review]

File: website/docs/r/is_lb_listener.html.markdown (hunk 2 above)
Comment (BShrivastav) [reply website/docs/r/is_lb_listener.html.markdown:190]: This is still has to be to support the CRNs of secrets from secrets manager.

File: website/docs/r/is_vpn_server.html.markdown (hunk 1 above)
Comment (BShrivastav) [reply website/docs/r/is_vpn_server.html.markdown:53]: Act as placeholder to support the CRNs of secrets from secrets manager.

**Review by sumitkumartiwari — APPROVED** [review]

**Comment by BShrivastav** [comment]
Raised upstream: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4310

**Pull request closed without merging**